deploy.py

```diff
@@ -1,24 +1,27 @@
-import os
 import argparse
+import os
 
 from bentoml.saved_bundle import load_bento_service_metadata
 
-from utils import (
-    get_configuration_value,
-    create_ecr_repository_if_not_exists,
-    console,
-)
-from aws_lambda import (
-    generate_lambda_deployable,
-    generate_lambda_resource_names,
-    generate_aws_lambda_cloudformation_template_file,
-    call_sam_command,
-)
+if __name__ == "__main__":  # use abs import if running as scripts
+    from aws_lambda import (call_sam_command,
+                            generate_aws_lambda_cloudformation_template_file,
+                            generate_lambda_deployable,
+                            generate_lambda_resource_names)
+    from utils import (console, create_ecr_repository_if_not_exists,
+                       get_configuration_value)
 
+else:
+    from .aws_lambda import (call_sam_command,
+                             generate_aws_lambda_cloudformation_template_file,
+                             generate_lambda_deployable,
+                             generate_lambda_resource_names)
+    from .utils import (console, create_ecr_repository_if_not_exists,
+                        get_configuration_value)
```

Comment:
@yubozhao this is one potential fix for the issue of relative imports v/s absolute imports. When calling as scripts (ie __name__ is '__main__') relative imports do not work since they don't have a parent module to relate from. but when the operator is importing the operator as a module it needs relative imports. Like is said this is one way to solve the issue and keep it consistent with the operator_template too. Let me know what you think about this approach.

Reply:
not doing it